pkg/apis/core/typeconfig/util.go

```diff
@@ -19,6 +19,7 @@ package typeconfig
 import (
 	"fmt"
 
+	"github.com/pkg/errors"
```

Comment:
Where no other 3rd party imports are present, please adhere to the following import formatting scheme:

```golang
// system imports

// 3rd party imports (including errors)

// kube imports
```